data/core/units/saurians/Prophet.cfg

```diff
@@ -0,0 +1,91 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Saurian Prophet
+    name= _ "Saurian Prophet"
+    race=lizard
+    gender=male
+    image="units/saurians/prophet/prophet.png"
+    profile="portraits/saurians/augur.png"
+    hitpoints=43
+    movement_type=lizard
+    movement=6
+    experience=150
+    level=3
+    alignment=chaotic
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=51
+    usage=archer
+    description= _ "Certain saurians are seen dressed in arcane regalia and covered head to toe in horrifying, esoteric shapes and markings, both with paint and tattoo. Some suspect them to be visionaries, or prophets amongst their ilk. But whatever ‘societal’ function they may have, they are undeniably powerful in the strange magics their kind possess, and are beings to be wary of if ever seen."
```

Comment:
```suggestion
    description= _ "Certain saurians are seen dressed in arcane regalia and covered head to toe in horrifying, esoteric shapes and markings, both with paint and tattoo. Some suspect them to be visionaries, or prophets amongst their ilk. But whatever ‘societal’ function they may have, they are undeniably powerful in the strange magics their kind possess."
```

Reply:
@nemaara [Oracle.cfg uses the same desciption](https://github.com/wesnoth/wesnoth/blob/master/data/core/units/saurians/Oracle.cfg#L18) so you may want to apply this change over there too.